### PR TITLE
Asset registry docs

### PR DESCRIPTION
**Pull request by MHA10**

### CODE REVIEW

**Comment by dockerymick** [comment]
@MHA10 thanks for creating your PR. This is not exactly how i imagined the docs. Is each file a different API? Based on my understanding, I believe each file is not its own API. It seems like each file (except the `asset_registry.md` file) represents the actions that can be done using the Asset registry API. 

I recommend that we update the `asset_registry.md` file by listing the actions and describing what can be done. Similar to how you are proposing, let's just move it to one file. I will add some suggestions

File: _pages/asset_registry.md [hunk 1]
```diff
@@ -5,6 +5,35 @@ permalink: /asset_registry
...
+* Verify that the S2 Geometry library is installed on your local environment. See [S2 Geometry Quick Start](https://s2geometry.io/devguide/cpp/quickstart).
+
+<!--Should the user have python installed to use AgStack? Will Python already be within the stack? -->
+
+
+## Search for an exisiting asset
+
+You can search for specific assets by entering the geospatial information for your polygon. After you query your polygon, you receive a 256-byte/16-Char alphanumeric unique ID. You can view only the assets that are available from your user account.
+  
+<!--Enter the following query to search for multiple assets-->
```
[the remark this answers is not in the document]
Comment (dockerymick) [reply _pages/asset_registry.md:26]: We can discuss more in the meeting

**Review by dockerymick — COMMENTED** [review]
Made some suggestions for asset registry doc and made some updates for the user registry docs. Let's discuss more during our live review

**Comment by dockerymick** [comment]
https://github.com/agstack/agstack-website/issues/14

File: _pages/user_registry.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* Log in to AgStack
+
+* Update user account settings
+
+* Search for domains
+
+* Search for authority token 
+
+* Log out of your user account
+
```
Comment (dockerymick) [inline _pages/user_registry.md:30]: @Aanish97 May you review this file. I am still working on edits

File: _pages/asset_registry.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dockerymick) [reply _pages/asset_registry.md:26]: @MHA10 ^

**Comment by MHA10** [comment]
> @MHA10 thanks for creating your PR. This is not exactly how i imagined the docs. Is each file a different API? Based on my understanding, I believe each file is not its own API. It seems like each file (except the `asset_registry.md` file) represents the actions that can be done using the Asset registry API.
> 
> I recommend that we update the `asset_registry.md` file by listing the actions and describing what can be done. Similar to how you are proposing, let's just move it to one file. I will add some suggestions

**Comment by MHA10** [comment]
> @MHA10 thanks for creating your PR. This is not exactly how i imagined the docs. Is each file a different API? Based on my understanding, I believe each file is not its own API. It seems like each file (except the `asset_registry.md` file) represents the actions that can be done using the Asset registry API.
> 
> I recommend that we update the `asset_registry.md` file by listing the actions and describing what can be done. Similar to how you are proposing, let's just move it to one file. I will add some suggestions

> @dockerymick , each .md file is a different API.

File: _pages/asset_registry.md [hunk 3]
```diff
@@ -5,6 +5,35 @@ permalink: /asset_registry
...
+
+
+<!--are these prerequisite accurate or will this library already be within the stack? If the library is a part of AgStack, how does the user access the library?
+* You must have a user account for AgStack.
+
+<!--If this prereq is required, what command should the user run to verify the installation?-->
+* Verify that the S2 Geometry library is installed on your local environment. See [S2 Geometry Quick Start](https://s2geometry.io/devguide/cpp/quickstart).
+
+<!--Should the user have python installed to use AgStack? Will Python already be within the stack? -->
+
```
[the remark this answers is not in the document]
Comment (MHA10) [reply _pages/asset_registry.md]: @dockerymick , these are independent functionalities and I don't believe these should be in Prerequisites.

File: _pages/asset_registry.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MHA10) [reply _pages/asset_registry.md]: @dockerymick, each .md file in apis/asset-registry represents how to query each API and the expected response.
Are you referring to it?

File: _pages/user_registry.md (hunk 2 above)
Comment (MHA10) [reply _pages/user_registry.md:30]: @dockerymick, seems good so far. Let me know if need any specific details.

Comment (MHA10) [reply _pages/user_registry.md:30]: @dockerymick, seems good so far. Let me know if needed any details.

**Comment by dockerymick** [comment]
Closing this because there are conflicts and we have made more progress. I am still working on updating the docs based on the info that you sent. I'll create a PR over the weekend

**Pull request closed without merging**